src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryText.cs

```diff
@@ -278,14 +278,14 @@ public static async ValueTask<FormulaValue> Coalesce(EvalVisitor runner, EvalVis
             return new BlankValue(irContext);
         }
 
-        public static FormulaValue Lower(IRContext irContext, StringValue[] args)
+        public static FormulaValue Lower(EvalVisitor runner, EvalVisitorContext context, IRContext irContext, StringValue[] args)
         {
-            return new StringValue(irContext, args[0].Value.ToLower());
```

Comment:
>ToLower [](http://example.com/codeflow?start=60&length=7)

Curious - is this the sort of thing that the Analyzers would have caught?

Reply:
It would.
Error	CA1304	The behavior of 'string.ToLower()' could vary based on the current user's locale settings...